Makefile

```diff
@@ -113,59 +113,64 @@ manifests-gen-crds: controller-gen yq
 	for YAMLFILE in config/crd/bases/actions*.yaml; do \
 		$(YQ) '.spec.preserveUnknownFields = false' --inplace "$$YAMLFILE" ; \
 	done
+	make manifests-gen-crds-fix DELETE_KEY=x-kubernetes-list-type
+	make manifests-gen-crds-fix DELETE_KEY=x-kubernetes-list-map-keys
```

Comment:
The new k8s.io dependency seems to have resulted in controller-gen additionally generating `x-kubernetes-list-map-keys` fields for pod specs embedded in CRDs.

This line removes the fields so that CRD validation passes.